tests/JWT.Tests.NETFramework/JWT.Tests.NETFramework.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>JWT.Tests.NETFramework</RootNamespace>
     <AssemblyName>JWT.Tests.NETFramework</AssemblyName>
-    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>

```

Comment:
Can you please explain this change?

Reply:
Oh yeah, you're right, Visual Studio 2019 upgraded my fork of the project, I'm going to remove it

Reply:
Cool! Can you try net473, would it work on the build server? I always love to be on the version, especially in tests.

Reply:
sure thing ! I'm doing it right now !

Reply:
Should be done !